src/lib/mina_generators/zkapp_command_generators.ml

```diff
@@ -234,13 +239,14 @@ let gen_account_precondition_from_account ?failure
     | _ ->
         return (Account_update.Account_precondition.Nonce nonce)
 
-let gen_fee (account : Account.t) =
+let gen_fee ~num_updates (account : Account.t) =
   let balance = account.balance in
-  let lo_fee = Mina_compile_config.minimum_user_command_fee in
-  let hi_fee =
+  let lo_fee =
     Option.value_exn
-      Currency.Fee.(scale Mina_compile_config.minimum_user_command_fee 2)
+      Currency.Fee.(
+        scale Mina_compile_config.minimum_user_command_fee (num_updates * 2))
```

Comment:
why is `num_updates` related to the fee?

Reply:
I was taking this change from deepthi.